toolkit/scss/shared_placeholders/_dm-typography.scss

```diff
@@ -1,10 +1,30 @@
 @import "_typography.scss";
+@import "_measurements.scss";
 
-%heading-xmedium {
+// All heading styles, except heading-xmedium, are copied from GOVUK Elements as of this commit:
```

Comment:
Sorry to be pedantic, Tom, but _I_ think it would better to also have another comment describing where the other header styles are defined -- sort of like the one was there before.

Maybe something like:
```css
// `heading-xlarge`, `heading-large`, `heading-medium`, and `heading-small` are inherited from GOVUK Elements as of this commit:
// https://github.com/alphagov/govuk_elements/commit/7aaa646a59019b6a90e8a88ddc81ac61950b9efe
```

(or whatever you think would be better).  

Personally, I didn't make the connection that the other headings were defined already in until  I read the other comment you deleted.

Should be good to go after that :)